### ISSUE TITLE
The browser chat and app chat window does not auto refresh and push notification is not working

### ISSUE THREAD

**Issue by nibircse** [jaullo/Snappy#15]
i have installed the server and have been able to connect the app with the server. the system works except

the old issue of push notification and auto refresh. 

there is not any instruction in this project so i followed the old spika instructions and it seems not working. 

any help will be very appreciated.
@jaullo @bafplus

**Comment by nibircse** [comment]
i also edited the asynctaskcontroller that you said to change.

**Comment by jaullo** [comment]
Hello @Yeasin-Ar-Rahman, in your server code u need to change the localhost validation and set your internal/public IP address or domain name.

Best regards

**Comment by nibircse** [comment]
i suppose you are referring to line number 42 and 112 in AsyncTaskController.php that is 

if($host != "localhost"){
                return $self->returnErrorResponse("invalid access to internal API");
            }

to 

if($host != "server-ip"){
                return $self->returnErrorResponse("invalid access to internal API");
            }

Thank you very much for your response.

**Comment by jaullo** [comment]
Yes that lined. Also live asynctaskcontroller without changes (don't make the changes i sayed) just change the ip addres
